netlify.toml

```diff
@@ -1302,4 +1302,22 @@ to = "/docs/api-reference/logistics-api"
 force = true
 from = "/docs/api-reference/pricing-api-overview"
 status = 308
-to = "/docs/api-reference/pricing-api"
\ No newline at end of file
+to = "/docs/api-reference/pricing-api"
+
+[[redirects]]
+force = true
+from = "/docs/recipes/development/vtex-io-cli-installation-and-command-reference/#command-reference"
```

Comment:
As far as I know, hashlinks (`#`) are not supported in the source slug ([ref](https://github.com/vtexdocs/dev-portal-content?tab=readme-ov-file#how-can-i-create-a-redirect)). Would it make sense to redirect the whole doc (`/docs/recipes/development/vtex-io-cli-installation-and-command-reference/` or only this section?